setup.py

```diff
@@ -64,7 +64,7 @@ def read(*parts):
     install_requires=install_requirements,
     extras_require={
         'tests': test_requirements,
-        'filer': ['django-filer', ],
+        'filer': ['django-filer==1.2.4', ],
```

Comment:
Could we then go with a range like "django-filer>=1.2.0,<1.3.0" ? I would be happy to merge the PR then :-)